src/ekore/operator_matrix_elements/unpolarized/space_like/as3/aqqNS.py

```diff
@@ -1,10 +1,15 @@
+"""The |N3LO| |OME| :math:`A_{qq}^{NS,(3)}`."""
+
 import numba as nb
 import numpy as np
 
+from .....harmonics.log_functions import lm11, lm12, lm13
+
 
 @nb.njit(cache=True)
-def A_qqNS(n, sx, nf, L):
-    r"""Computes the |N3LO| singlet |OME| :math:`A_{qq}^{NS,(3)}(N)`.
+def A_qqNS(n, sx, nf, L, eta):
```

Comment:
Mmm in #202 we said that `is_singlet` (and likewise `eta`) should not be exposed outside the function - why are you doing here?

Reply:
No this is correct but actually the doc string is not correct. I believe this contribution is  the analogous of `ns,+` so it appears both in singlet and non-singlet quantities.